ethexe/common/src/network.rs

```diff
@@ -51,6 +53,7 @@ pub enum SignedValidatorMessage {
     ProducerBlock(SignedData<ValidatorAnnounce>),
     RequestBatchValidation(SignedData<ValidatorRequest>),
     ApproveBatch(SignedData<ValidatorReply>),
+    InjectedPromise(SignedData<ValidatorInjectedPromise>),
```

Comment:
We don't have any other promises, and inner data could be renamed as well 
```suggestion
    Promise(SignedData<ValidatorInjectedPromise>),
```